src/components/Branding/index.ts

```diff
@@ -0,0 +1,3 @@
+export { ReactComponent as LogoLarge } from './images/logo_large.svg';
+export { ReactComponent as LogoMaximized } from './images/logo_maximized.svg';
+export { ReactComponent as LogoSmall } from './images/logo_small.svg';
```

Comment:
Nit:  I'd consider renaming this from `index.ts` to something else.  Perhaps `raw-icons.ts`.   It think it could be confusing as `index.ts` is meant to be the export point for a directory, but unless I am mistaken, these are not meant to be exported publically as much as it is just intermediary to get to `LogoBrand`, `LogoText`, and `Logo`, right?

Reply:
You're correct, it's just an intermediary to get to `LogoBrand`, `LogoText`, and `Logo`.  I've updated `index.ts` to be named `raw-icons.tx`.